home_work_7/mod_menu.py

```diff
@@ -0,0 +1,10 @@
+def menu():
+    print("\n\t\t\tМеню программы\n\n", flush=False)
+    print("1. Добавить новый контакт")
+    print("2. Удалить существующий контакт")
+    print("3. Удалить все контакты")
+    print("4. Поиск контакта")
+    print("5. Показать все контакты")
+    print("6. Выход из телефонной книги")
+    choice = int(input("Введите пункт меню: "))
```

Comment:
Зачем столько вызовов функций принт, когда можно все это сделать в функции `input`?
```suggestion
    choice = int(input(
        "\n\t\t\tМеню программы\n\n\n"
        "1. Добавить новый контакт\n"
        "2. Удалить существующий контакт\n"
        "3. Удалить все контакты\n"
        "4. Поиск контакта\n"
        "5. Показать все контакты\n"
        "6. Выход из телефонной книги\n"
        "Введите пункт меню: "
    ))
```
Как то так.